make/DynamoRIOConfig.cmake.in

```diff
@@ -1109,7 +1109,15 @@ function (configure_DynamoRIO_static target)
   # We need libc before the ntdll_imports pulled in for DR.
   _DR_get_static_libc_list(static_libc)
   target_link_libraries(${target} ${static_libc})
-  target_link_libraries(${target} dynamorio_static)
+  # For the weakly linked routines defined in drlibc, we want to use DR's
+  # definitions when available. So we use dynamorio_static_nohide, which does
+  # not hide its symbols, for building static binaries.
```

Comment:
Comment on the downsides: pushes our hundreds of global symbols into the app's namespace (we have renamed some of the more-likey-to-collide ones as part of #3348).

Reply:
Done.